frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/components/assisted-installer/DetailsForm.tsx

```diff
@@ -108,9 +108,19 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ control, handleChange, contro
         }
         control.summary = () => {
             return Object.keys(fields).map((key) => {
+                let desc = get(control, `active.${key}`);
+                if (key === "openshiftVersion") {
+                    const imageSet = clusterImageSets.find((cis) => cis.metadata.name === get(control, `active.${key}`))
+                    const releaseImage = imageSet?.spec?.releaseImage
+                    const tagStartIndex = releaseImage?.indexOf(':') ?? 0
+                    desc = releaseImage?.slice(
+                        tagStartIndex + 1,
+                        releaseImage.indexOf('-', tagStartIndex)
+                    )
```

Comment:
we should use the same logic of getting the version as in [AI lib](https://github.com/openshift-assisted/assisted-ui-lib/blob/5edb3447fc5ed50c878188b59ab8fe14d634d783/src/cim/components/helpers/versions.ts#L5).